test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorResolverTest.cs

```diff
@@ -25,9 +25,14 @@ private static TagHelperDescriptor Valid_PlainTagHelperDescriptor
         {
             get
             {
-                return new TagHelperDescriptor("valid_plain",
-                                               Valid_PlainTagHelperType.FullName,
-                                               AssemblyName);
+                return new TagHelperDescriptor(
+                    prefix:  string.Empty,
+                    tagName: "valid_plain",
+                    typeName: Valid_PlainTagHelperType.FullName,
+                    assemblyName: AssemblyName,
+                    attributes: Enumerable.Empty<TagHelperAttributeDescriptor>(),
+                    requiredAttributes: Enumerable.Empty<string>(),
+                    designTimeDescriptor: null);
```

Comment:
Ick. These constructor overloads exist in part to avoid the extra verbiage in our tests.